wording clarifications (#3)
MYOB-Technology/ops-technical-test: README.md

]
```

> - tests or a test suite.s

Reviewer: Typo

Author: thanks

## Fit and Finish

Once the application has been written, continue with the following additions:

- provide a means of packaging your application as a single deployable artifact which encapsulates its dependencies

> - create a pipeline that builds your application on each commit; Travis, Github Actions or similar

Reviewer: ```suggestion
- create a pipeline that builds your application on each commit; Travis, Github Actions, or similar
```

- describe or demonstrate any risks associated with your application/deployment

> - write a clear and understandable `README` which explains your application, its packaging steps, and how it meets the requirements

Reviewer 2: and how to run it

> - write a clear and understandable `README` which explains your application and its packaging steps, as well as how it meets requirements

Reviewer: ```suggestion
- write a clear and understandable `README` which explains your application, its packaging steps, and how it meets the requirements
```
